### ISSUE TITLE
Migrate away from Bower

### ISSUE THREAD

**Issue by bradfrost** [pattern-lab/styleguidekit-assets-default#76]
What to do about this? https://bower.io/blog/2017/how-to-migrate-away-from-bower/

Cc @bmuenzenmeyer

**Comment by bmuenzenmeyer** [comment]
This factors very much into the larger scale effort. For now I'd focus on getting your design refactor done, then we can focus on some of the behind the scenes work that makes the machine run.

**Comment by bradfrost** [comment]
@bmuenzenmeyer Yep I figured as much. Just flagging it since I just read the post. 👍

**Comment by sghoweri** [comment]
@bmuenzenmeyer @bradfrost @EvanLovely FWIW, I [already killed off bower](https://github.com/drupal-pattern-lab/styleguidekit-assets-default/blob/feature/modernize-patternlab-js/package.json) in the refactor I started working on 3 months ago (in addition to chipping away at getting the JS to [compile with Webpack](https://github.com/drupal-pattern-lab/styleguidekit-assets-default/blob/feature/modernize-patternlab-js/webpack.config.js#L15), breaking apart the JS into more [standalone modules](https://github.com/drupal-pattern-lab/styleguidekit-assets-default/tree/feature/modernize-patternlab-js/src/scripts) that manage their own includes without a full JS refactor (which we really do need), getting all the JS script tags in PL to [load up asynchronously ](https://github.com/drupal-pattern-lab/styleguidekit-assets-default/blob/feature/modernize-patternlab-js/src/html/index.html#L108)without being dependent on a particular loading order, etc).

https://github.com/drupal-pattern-lab/styleguidekit-assets-default/tree/feature/modernize-patternlab-js

There's just a couple minor things that I never got around to on the Drupal Pattern Lab fork (namely just cleaning up some of the bower configs that are no longer used + reverting the hogan.js templating for the navigation since this branch also includes me getting the main PL menu to render with Twig templating](https://github.com/drupal-pattern-lab/styleguidekit-assets-default/blob/feature/modernize-patternlab-js/src/scripts/modules/layout/index.js) vs Mustache).

**Comment by bmuenzenmeyer** [comment]
Sounds cool @sghoweri - sounds like a trail is already blazed by your hard work !

I think a good time to tackle this is once other features settle in - this is mostly backend convenience and doesn't affect users much

**Comment by sghoweri** [comment]
@bmuenzenmeyer @EvanLovely trying to get my ducks in a row on this (busy week!) but for what it’s worth I think I’ve got the work needed to be done on this already figured out for a while now, just need to get the PR specific bits to this put together (and likely fold in the build tools updates at the same time given the overlapping dependencies):

https://github.com/bolt-design-system/bolt/tree/epic/v2/packages/styleguidekit-assets-bolt
